op-chain-ops/solc/types.go

```diff
@@ -158,6 +158,8 @@ type AstNode struct {
 	StateMutability  string            `json:"stateMutability,omitempty"`
 	Virtual          bool              `json:"virtual,omitempty"`
 	Visibility       string            `json:"visibility,omitempty"`
+	Modifiers        []AstNode         `json:"modifiers,omitempty"`
```

Comment:
Can undo this diff now

Reply:
But this is still needed though, maybe in the future, no?

Reply:
I think let's still remove it for now, we can add it when it becomes relevant